src/index.ts

```diff
@@ -1,15 +1,32 @@
 console.clear();
 
-let personas = [12, 36, 18, 40, 8, 32, 46, 22, 25, 39, 18, 19];
+let personas: number[] = new Array(270);
+let edadMinima: number = 18;
+let edadMaxima: number = 40;
 
-function esMayorQue21(numero) {
-  if (numero > 21) {
-    console.log(numero + " es mayor que 21.");
+let cantidadMenores21: number = 0;
+let cantidadMayores21: number = 0;
+
+for (let indice = 0; indice < 270; indice++) {
+  let numeroAleatorio = Math.floor(
+    Math.random() * (edadMaxima - edadMinima) + edadMinima
+  );
+  personas[indice] = numeroAleatorio;
+}
```

Comment:
Hacemos un **for**:
- Este **for** es un ciclo que se repite 270 veces, como el largo del arreglo.
- En cada vuelta lo que hace el guardar un valor aleatorio entre 18 (edadMinima) y 40 (edadMaxima) en la posición del arreglo correspondiente a esa vuelta `(personas[indice]).`
- Un valor aleatorio entre una **edadMinima** y una **edadMaxima** se calcula asi: `Math.floor(Math.random() * (edadMaxima - edadMinima) + edadMinima);`